src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,11 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball firstBall = Lottery.getRandomBall();
```

Comment:
The `getRandomBall` method is being called statically from the `Lottery` class, which is against the checklist's guidance to avoid static methods. Instead, create an instance of the `Lottery` class and call the method on that instance.